src/test/java/victor_murashev/step_definitions/OrderFormSteps.java

```diff
@@ -24,48 +24,32 @@ public class OrderFormSteps {
     @Given("the user is on the {string}")
     public void theUserIsOnThe(String url) {
         driver.get(url);
+        WebDriverHooks.pauseExecution(500);
```

Comment:
pauseExecution is unknown method 
and by the way you can make static import 
to replace `WebDriverHooks.pauseExecution(500); `on just `pauseExecution(500);`

Reply:
@dgrimaliuc-crunchyroll I moved all intermediate methods like start/close browser, setting delays into WebDriverHooks-class just to avoid code duplication in main classes

Reply:
the WebDriverHooks is not present here in PR
So it will broke code build for everyone

Reply:
@dgrimaliuc-crunchyroll agreed< I just forget to include the changes into current PR. Check now: the updated WebDriverHooks has been included here.